content/en/continuous_integration/guides/flaky_test_management.md

```diff
@@ -51,10 +60,20 @@ Click on the **New Flaky** number and then click **Ignore flaky tests**.
 
 ## Watch for known flaky failed tests
 
+Known flaky failed tests are tests that have flaky behavior on the current or default branch of the repository.
+
+### In the Test Runs page
+
+1. In the facets list on the left sidebar, in the "Test" section, expand the "Known Flaky" facet and check `true`
+2. Failed test runs that were known to be flaky as per the definition above are displayed.
+
+
+### In the Branches page
+
 1. On the Tests page, select the **Branches** view.
 
 2. Filter the table to see any branches, services, or commits of interest.
 
-3. The **Failed** column contains the number of failed tests and known flaky failed tests in the latest commit. Known flaky failed tests are tests that have flaky behavior on the current or default branch of the repository.
+3. The **Failed** column contains the number of failed tests and known flaky failed tests in the latest commit.
 
 {{< img src="ci/known-flaky-failed-tests.png" alt="CI Tests Branches view with a branch selected and a text box in the Failed column displaying 1 tests failed and 1 known flaky" style="width:100%;">}}
```

Comment:
```suggestion
{{< img src="ci/known-flaky-failed-tests.png" alt="CI Tests Branches view with a branch selected and a text box in the Failed column displaying 1 tests failed and 1 known flaky" style="width:100%;">}}

[1]: https://app.datadoghq.com/ci/test-runs
[2]: https://app.datadoghq.com/ci/test-services
```

Reply:
should the app URL differ depending on the chosen datacenter?